lib/Interpreter/CppInterOp.cpp

```diff
@@ -2463,6 +2463,10 @@ namespace Cpp {
     std::vector<const char *> ClingArgv = {"-resource-dir", ResourceDir.c_str(),
                                            "-std=c++14"};
     ClingArgv.insert(ClingArgv.begin(), MainExecutableName.c_str());
+#ifdef _WIN32
+    // FIXME : Workaround Sema::PushDeclContext assert on windows
+    ClingArgv.push_back("-fno-delayed-template-parsing");
```

Comment:
Do we need `-femulated-tls` for the TLS related issues? We do this here in cling:

https://github.com/root-project/root/blob/fda4630190de40d817b850efa240c4ed72a20b8e/interpreter/cling/lib/Interpreter/IncrementalJIT.cpp#L388-L390

Reply:
I tried -fno-emulated-tls there and still got the same emulated TLS symbol errors. It looks like its defaulted to on from JITTargetMachineBuilder's constructor.

Reply:
Do you think we need a patch in upstream clang?

Reply:
Maybe I don't really know how it should work for windows.